cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -43,13 +45,19 @@ class RandomAccessObject : public arrow::io::RandomAccessFile {
     chunks_ = std::vector<ceph::bufferlist*>();
   }
 
+  /// \brief Check if the file stream is closed.
+  /// \return Status. OK if the stream is not closed.
   arrow::Status CheckClosed() const {
     if (closed_) {
       return arrow::Status::Invalid("Operation on closed stream");
     }
     return arrow::Status::OK();
   }
 
+  /// \brief Check if the position of the object is valid.
+  /// \param[in] position The position to check.
+  /// \param[in] action The action to perform on the position.
+  /// \return Status. OK if the position is valid.
   arrow::Status CheckPosition(int64_t position, const char* action) const {
```

Comment:
We need not document this

Reply:
Done. I have made this and the others into a comment.